include/deal.II/base/quadrature_point_data.h

```diff
@@ -729,21 +743,27 @@ pack_cell_data(const CellIteratorType &                           cell,
     std::is_base_of<TransferableQuadraturePointData, DataType>::value,
     "User's DataType class should be derived from QPData");
 
+  if (!data_storage->stores_data(cell))
+    {
+      matrix_data = FullMatrix<double>();
```

Comment:
```suggestion
      matrix_data.clear();
```

Reply:
I couldn't find the `clear()` method in the `FullMatrix<Number>` interface. I used the `reinit({0, 0})` method instead.

Reply:
Yes, that makes sense.